.github/workflows/ci.yaml

```diff
@@ -84,11 +84,15 @@ jobs:
           - path: source-mysql
             additionalTags:
               - source-mariadb
+              - source-amazonaurora
           - path: source-postgres
             additionalTags:
               - source-alloydb
+              - source-amazonaurora
```

Comment:
These two source connectors will collide with each other. I would suggest we name them like this so we can build two separate images:

```
source-amazon-aurora-mysql
source-amazon-aurora-postgres
```